junit5-extension/src/test/java/io/kroxylicious/testing/kafka/junit5ext/ParameterExtensionTest.java

```diff
@@ -34,17 +36,20 @@
 import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.testcontainers.shaded.org.awaitility.Awaitility.await;
 
 @ExtendWith(KafkaClusterExtension.class)
 public class ParameterExtensionTest extends AbstractExtensionTest {
 
+    private static final Duration NODE_TIMEOUT = Duration.ofSeconds(5);
+
     @Test
     public void clusterParameter(@BrokerCluster(numBrokers = 2) KafkaCluster cluster)
             throws ExecutionException, InterruptedException {
+        await().atMost(5, TimeUnit.SECONDS).untilAsserted(() -> assertEquals(2, describeCluster(cluster.getKafkaClientConfiguration()).nodes().get().size()));
```

Comment:
Why don't you use the same approach than in the other tests?
`await().atMost(NODE_TIMEOUT).untilAsserted(...)`

Reply:
that was unintentional.